bigchaindb/backend/mongodb/connection.py

```diff
@@ -129,6 +206,10 @@ def initialize_replica_set(host, port, connection_timeout, dbname, ssl, login, p
     else:
         _wait_for_replica_set_initialization(conn)
         logger.info('Initialized replica set')
+    finally:
+        if conn is not None:
+            logger.info('Closing initial connection to MongoDB')
+            conn.close()
```

Comment:
I think this was a nascent bug in the module; we weren't closing the connection here.